src/main/java/homework_2/RandomCharsTable.java

```diff
@@ -0,0 +1,65 @@
+package homework_2;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class RandomCharsTable {
+    private static char[][] createRandomTable(int x, int y){
+        char[][] table = new char[x][y];
+        Random random = new Random();
+        for (int i = 0; i < x; i++){
+            for (int j = 0; j < y; j++){
+                table[i][j] = (char) (random.nextInt((90 - 65)+1)  + 65);
```

Comment:
Opt: i'm not sure these mathematical operations need to be done each char generation. Possible performance issue on big tables